### PR TITLE
[docs] Update how-to-contribute.md

### PR DESCRIPTION
**Pull request by zetwhite**
add ONE-DCO wiki page link to contribution guide

ONE-DCO-1.0-Signed-off-by: SeungHui Youn <sseung.youn@samsung.com>

### CODE REVIEW

File: docs/howto/how-to-contribute.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This section explains the steps to create a pull request (PR).
 1. Create a commit
 
    It is time to create a commit for submission once you are convinced that your contribution is
-   ready to go. Please include signed-off message at the end of commit message. If not, your pull
+   ready to go. Please include [signed-off message](https://github.com/Samsung/ONE/wiki/ONE-Developer's-Certificate-of-Origin) at the end of commit message. If not, your pull
```
Comment (seanshpark) [inline docs/howto/how-to-contribute.md:37]: How about begin a new line at `at the end...` not to make this line too long?

Comment (zetwhite) [reply docs/howto/how-to-contribute.md]: good idea :+1:

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Review by llFreetimell — APPROVED** [review]
LGTM!

**Review by llFreetimell — APPROVED** [review]
LGTM!

**Review by lemmaa — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**